### PR TITLE
Migration ef6

### PR DESCRIPTION
**Pull request by Saisang**

### CODE REVIEW

**Comment by bradygaster** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/aspnet/EntityFramework.Docs, working directory: W:\mmk2-s, target branch: master, target branch commit: 190727325582d4f4198f57e9b14d0df6f8889717, source branch: migration_ef6, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C9%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201708091810105175-439%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bradygaster** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/aspnet/EntityFramework.Docs, working directory: W:\hkf4-s, target branch: master, target branch commit: 190727325582d4f4198f57e9b14d0df6f8889717, source branch: migration_ef6, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C9%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201708091816577644-439%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by bradygaster** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[entity-framework/ef6/case-studies-for-ado-net-entity-framework.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/case-studies-for-ado-net-entity-framework.md) | :warning:Warning | | [Details](#user-content-entity-framework%2Fef6%2Fcase-studies-for-ado-net-entity-framework.md)
[entity-framework/ef6/introduction-to-entity-framework.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/introduction-to-entity-framework.md) | :warning:Warning | | [Details](#user-content-entity-framework%2Fef6%2Fintroduction-to-entity-framework.md)
[.localization-config](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/.localization-config) | :white_check_mark:Succeeded | |
[entity-framework/ef6/StructuredTOC.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/StructuredTOC.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/ado-net-entity-framework-articles.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/ado-net-entity-framework-articles.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/ado-net-entity-framework-tools.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/ado-net-entity-framework-tools.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/building-an-mvc-3-app-with-code-first-and-entity-framework-4-11.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/building-an-mvc-3-app-with-code-first-and-entity-framework-4-11.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/building-an-mvc-3-app-with-code-first-and-entity-framework-4-12.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/building-an-mvc-3-app-with-code-first-and-entity-framework-4-12.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/building-an-mvc-3-app-with-database-first-and-entity-framework-4-11.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/building-an-mvc-3-app-with-database-first-and-entity-framework-4-11.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/building-an-mvc-3-app-with-database-first-and-entity-framework-4-12.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/building-an-mvc-3-app-with-database-first-and-entity-framework-4-12.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/building-an-mvc-3-app-with-model-first-and-entity-framework-4-11.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/building-an-mvc-3-app-with-model-first-and-entity-framework-4-11.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/building-an-mvc-3-app-with-model-first-and-entity-framework-4-12.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/building-an-mvc-3-app-with-model-first-and-entity-framework-4-12.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/code-first-dataannotations1.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/code-first-dataannotations1.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/code-first-dataannotations2.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/code-first-dataannotations2.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/code-first-mapping-fluent-api.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/code-first-mapping-fluent-api.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/code-first-relationship-fluent-api.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/code-first-relationship-fluent-api.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/code-first-relationships-fluent-api.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/code-first-relationships-fluent-api.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/community-blog-posts.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/community-blog-posts.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/consuming-an-entity-data-model-from-a-separate-net-project.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/consuming-an-entity-data-model-from-a-separate-net-project.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/creating-an-entity-data-model-from-a-database.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/creating-an-entity-data-model-from-a-database.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/creating-an-entity-data-model-the-model-first-way.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/creating-an-entity-data-model-the-model-first-way.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/creating-master-details-windows-forms-with-the-entity-framework.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/creating-master-details-windows-forms-with-the-entity-framework.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/customizing-code-first-to-an-existing-database.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/customizing-code-first-to-an-existing-database.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/data-binding-with-wpf-and-the-entity-framework.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/data-binding-with-wpf-and-the-entity-framework.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/database-initializers.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/database-initializers.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/drag-and-drop-databinding-with-the-entity-framework-and-wpf1.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/drag-and-drop-databinding-with-the-entity-framework-and-wpf1.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/drag-and-drop-databinding-with-the-entity-framework-and-wpf2.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/drag-and-drop-databinding-with-the-entity-framework-and-wpf2.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/edm-conceptual-storage-and-mapping-layers-and-their-relationships.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/edm-conceptual-storage-and-mapping-layers-and-their-relationships.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/edm-custom-mapping-with-ssdl.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/edm-custom-mapping-with-ssdl.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/edm-inheritance-mapping-scenarios.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/edm-inheritance-mapping-scenarios.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/edm-modeling-complex-types.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/edm-modeling-complex-types.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/edm-splitting-scenarios.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/edm-splitting-scenarios.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/edm-video-include.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/edm-video-include.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/edm-video-sidebar.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/edm-video-sidebar.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/ef-links.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/ef-links.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/ef-nav.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/ef-nav.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/ef-resources.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/ef-resources.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/ef6-runtime-alpha-license.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/ef6-runtime-alpha-license.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/ef6-runtime-beta-rc-license.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/ef6-runtime-beta-rc-license.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-4-1-validation.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-4-1-validation.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license-chs.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license-chs.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license-cht.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license-cht.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license-deu.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license-deu.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license-esn.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license-esn.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license-fra.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license-fra.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license-ita.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license-ita.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license-jpn.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license-jpn.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license-kor.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license-kor.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license-rus.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license-rus.md) | :white_check_mark:Succeeded | |
[entity-framework/ef6/entity-framework-5-license.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/entity-framework-5-license.md) | :white_check_mark:Succeeded | |

This comment lists only the first 50 files in the pull request.
<a name="entity-framework%2Fef6%2Fcase-studies-for-ado-net-entity-framework.md"></a>
### [entity-framework/ef6/case-studies-for-ado-net-entity-framework.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/case-studies-for-ado-net-entity-framework.md)
 - **Line 14**: **[Warning]** Invalid file link:(~/ef6/Entity Framework.md).

<a name="entity-framework%2Fef6%2Fintroduction-to-entity-framework.md"></a>
### [entity-framework/ef6/introduction-to-entity-framework.md](https://github.com/aspnet/EntityFramework.Docs/blob/migration_ef6/entity-framework/ef6/introduction-to-entity-framework.md)
 - **Line 20**: **[Warning]** Invalid file link:(~/ef6/Get Entity Framework.md).
 - **Line 23**: **[Warning]** Invalid file link:(~/ef6/Entity Framework (EF) Documentation.md).
 - **Line 27**: **[Warning]** Invalid file link:(~/ef6/Get Help Using Entity Framework.md).
 - **Line 37**: **[Warning]** Invalid file link:(~/ef6/Entity Framework Development Workflows Video.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C8%5C9%5Cc1705ea6-142e-cb02-f3b5-8dd98c6d1733%5CPullRequest%5C201708091820516295-439%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: .localization-config [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (divega) [inline .localization-config]: This seems to be a new file in the root. Is that intentional? Does it affect the EF Content as well? 

Just checking, I am not familiar with the details of the docs system.

**Comment by divega** [comment]
@Saisang I believe things look good although it is a bit hard to check until I have merged the ToC into the main one. Thanks for doing the migration. 

I will let you know if I run into anything.

**Comment by Saisang** [comment]
I've signed.

**Comment by Rick-Anderson** [comment]
@Saisang  is this the migration of EF6 from MSDN to docs.microsoft.com?
Has this been reviewed on review.docs.microsoft.com?

**Comment by divega** [comment]
Superseded by #624.

**Pull request closed without merging**